### ISSUE TITLE
[Bug] Block Targeting Treasure Hunt Enemies crash issue

### ISSUE THREAD

**Issue by Crescr** [Caraxi/SimpleTweaksPlugin#449]
When the function is turned on, the dialog box with the following error appears and the application is immediately terminated.
"Unknown Hard Error" >> Logged in Event Viewer -> System.

The following error is recorded in Event Viewer >> Application.
Application: ffxiv_dx11.exe
CoreCLR Version: 7.0.22.51805
.NET Version: 7.0.0
Description: The process was terminated due to an internal error in the .NET Runtime at IP 0x00007FFDAA7A63B5 (0x00007FFDAA540000) with exit code 0x80131506.

There are no errors recorded in the dalamud logs.

If you end up with this error, it will crash 100% immediately after loading your character unless you turn it off in the dalamud settings on the title screen.

**Comment by Caraxi** [comment]
That should not be possible

**Comment by Crescr** [comment]
I had some time, so I did a detailed investigation.

The problem occurs only when the following plugins are "enabled".
Rotation Solver
https://github.com/ArchiDog1998/RotationSolver

For the time being, I confirmed that the following works without any errors.
○RotationSolver - Disable → Treasure ~ - Enable
○ Treasure - Disable → RotationSolver - Enable

I'm guessing the RotationSolver is affecting this functionality by targeting mobs with the shortest distance.
It may be a problem on the RotationSolver side, but I will report the results for the time being.
